README.md

```diff
@@ -152,6 +152,24 @@ any previously configured value.
 | 2.3.1   | ^2.6.2                | *<br/>dev:0.59.8     | [AI master](https://github.com/microsoft/ApplicationInsights-JS/tree/master/extensions/
 | 2.3.0   | ^2.6.0                | *<br/>dev:0.59.8     | [AI master](https://github.com/microsoft/ApplicationInsights-JS/tree/master/extensions/
 
+## Nightly Builds
+
+To aid with testing and validation we also produce and publish nightly builds whenever there is a change from the previous build. These builds are published to the [NpmJs registry](https://www.npmjs.com/package/@microsoft/applicationinsights-react-native) on a successful build / test pass.
+
+This process also [tags the source code](https://github.com/microsoft/applicationInsights-react-native/tags) so that we can track the specific changes included using a nightly build specific version number which is the format "nightly-yymm-##" eg. ```nightly-2208-02```
```

Comment:
maybe "nightly.yymm-##" eg. ```nightly.2208-02```

Reply:
Nope, the tag has the "-" as the dot caused issues and has implications for semver (from memory)